roiextractors/tests/test_extractors.py

```diff
@@ -0,0 +1,46 @@
+import h5py
+import numpy as np
+import os
+import sys
+import unittest
+from ..example_datasets.toy_example import toy_example
+
+
+class TestSegmentationExtractors(unittest.TestCase):
```

Comment:
```suggestion
class TestExtractors(unittest.TestCase):
```

These should also include ImagingExtractors, so we generate the toy example only once for both